fix: AT1-72 ballot forms candidate preview & duties modals and others

Andrea-Villalobos opened the pull request:
- fixed modals not showing up such as duties and responsibilities & candidate preview when an abstain option is selected
- separated scripts; added js files
- some ui changes such as the radio button  border color and fill, paddings, a href style
- fixed footer in end point; when zoomed out, still sticks at the bottom

![image](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/162780943/c482f546-92a8-4fa3-9315-5c06e3b9975d)

Attachment: sample sql file
[sampledb.txt](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/files/15177721/sampledb.txt)

Ivan-Edan commented:
The endpoint footer doesn't stick at the bottom when I try to use this screen size:
![image](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/162779484/31d710e9-266f-4d44-b7fd-43cf1227209e)

I noticed that this part of the responsiveness the card is too big in this screen size. Please use proper padding and align/justify the text inside the card.
![image](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/162779484/7bdc29e7-c838-4233-be0f-b88bf2712829)

This part doesn't align properly.
![image](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/162779484/4ed22431-738d-46e1-ae96-ab60976cc6cf)

The button is a bit awkward from my perspective (idk if it is just my monitor size): 
![image](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/162779484/fba13101-7b69-437e-a0cd-89055224bfce)

The preview function is not centered enough and it appears at the top of my screen(P.S ignore the warning sign on my screen): 
![image](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/162779484/9e94cde6-a3fb-4376-9d7c-984aa9313c00)

The responsiveness of the end-point image doesn't match the figma. please adjust the margin/padding in the image.
![image](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/162779484/1cea88e2-e7d8-4dba-ab0a-f083a3c7915c)


About the module's back-end I cant really test it since it isn't my forte. Will wait for other reviewer to test that part

Ivan-Edan reviewed (changes requested):
Refer to my recent comment. The review is all about front-end of the modules. Wait for other reviewer to test its back-end

Andrea-Villalobos commented:
![Screenshot 2024-05-02 191035](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/162780943/d61d017f-56e7-42a5-9bd2-ad6e2777a48d)

- Tried to adjust the footer but when inspected, it does not stick to the bottom. 
- Will try to fix it in the next pr.

![Screenshot 2024-05-02 191341](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/162780943/765e229c-926e-4782-8b12-d9d6591566d6)

- Centered the text vertically and removed the submit button and reset

![Screenshot 2024-05-02 190715](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/162780943/d528373e-46a0-4753-8898-ac8c821bbabf)

-Modal is centered vertically and will add overlay in the next pr.

![Screenshot 2024-05-02 190936](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/162780943/7a26b408-723e-4767-9441-6797e99ca56b)

- Added padding.
- Will improve the layout in the next pr.

Additional:
Still trying to adjust the width of welcome modal responsiveness in mobile devices. Asking for assistance. Thank you.
